src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractCreatePdpImportServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationProcessingService;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractCreatePdpImportService;
+import edu.cornell.kfs.concur.service.ConcurRequestExtractFileService;
+
+public class ConcurRequestExtractCreatePdpImportServiceImpl implements ConcurRequestExtractCreatePdpImportService {
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractCreatePdpImportServiceImpl.class);
+	protected ConcurRequestExtractFileService concurRequestExtractFileService;
+
+	@Override
+    public void convertConcurRequestExtractsToPdpImports() {
```

Comment:
I like your method name better. The only caveat is there could be more than one Request Extract file that needs to be converted per batch processing run and your suggestion implies one file is being converted. Should that method name be adjusted to  createPdpFeedsFromRequestExtracts to reflect that multiple file conversion possibility?